scss/_variables.scss

```diff
@@ -1650,6 +1654,8 @@ $pagination-dark-active-item-border-color:  $brand-orange !default;
 $pagination-dark-disabled-bg:               $black !default;
 $pagination-dark-disabled-color:            $gray-700 !default;
 $pagination-dark-disabled-border-color:     $gray-700 !default;
+
+$pagination-item-max-width:                 2.5rem !default;
```

Comment:
Needed or not needed? Or needed only on mobile? It may depend of what you have done for the responsive pagination.

Reply:
I think we should take care about this kind of thing once one PR is merged. So let's keep it like that

Two things about this: Maybe group with others before dark variant variables + replace value by `$spacer * 2` could be easier to understand (if not you can resolve this) ?